queue/src/fiskaltrust.Middleware.Localization.QueueDE/SignatureFactoryDE.cs

```diff
@@ -134,7 +134,8 @@ public List<SignaturItem> GetSignaturesForPosReceiptTransaction(string startTran
                 CreateBase64Signature(SignatureTypesDE.StartTransactionResult, "start-transaction-signature", startTransactionSignatureBase64, true),
                 CreateBase64Signature(SignatureTypesDE.FinishTransactionPayload, "finish-transaction-payload", finishResultResponse.ProcessDataBase64, true),
                 CreateBase64Signature(SignatureTypesDE.FinishTransactionResult, "finish-transaction-signature", finishResultResponse.SignatureData.SignatureBase64, true),
-                CreateTextSignature(SignatureTypesDE.CertificationId, "<certification-id>", certificationIdentification, false) //TODO clarify
+                CreateTextSignature(SignatureTypesDE.CertificationId, "<certification-id>", certificationIdentification, false), //TODO clarify
+                CreateTextSignature(SignatureTypesDE.TseSerialNumber, "<tse-seriennummer>", finishResultResponse.TseSerialNumberOctet, true)
```

Comment:
@saschaknapp do you know if the TseSerialnumber has to be included in the QR Code too?

Reply:
@StefanKert The serial numbers shouldn't be included in the QR-Code, because according to the DSFinV-K 2.3 there is no space for it and they rather have the public key included:

Hinweis: Um derzeit eine Belegprüfung durchführen zu können, muss der Public-Key mit
im QR-Code enthalten sein. **Die Seriennummer des TSE entfällt dabei aus Platzgründen**
(kann für die Validierung aus dem Public-Key errechnet werden (SHA256 als OctetString))
DSFinV-K 2.3 Page 122